src/sidecar_comms/form_cells/base.py

```diff
@@ -98,7 +99,14 @@ def _sync_sidecar(self, change: Change):
         """Send a comm_msg to the sidecar to update the form cell metadata."""
         # not sending `change` through because we're doing a full replace
         # based on the latest state of the model
-        self._comm.send(handler="update_form_cell", body=self.dict())
+        data = self.dict()
+
+        if self.input_type == "datetime":
```

Comment:
Can this be moved into specific `FormCell` subclasses? Maybe `DateTime` overrides `_sync_sidecar` ?  Could also look into whether applying custom serializers are an option here, but I think those only effect `.json()` not `.dict()` so it might too much of a change

Reply:
Yeah, those only affect `.json()` - I like the `Datetime` method override idea, will go with that 👍